src/main/kotlin/com/launchdarkly/intellij/LDDocumentationProvider.kt

```diff
@@ -1,167 +0,0 @@
-package com.launchdarkly.intellij
```

Comment:
This is moved and renamed to `src/main/kotlin/com/launchdarkly/intellij/hover/HoverDocumentationProvider.kt`